docs/source/nodes_and_pipelines/pipeline_registry.md

```diff
@@ -0,0 +1,73 @@
+# The pipeline registry
```

Comment:
```suggestion
# Pipeline registry
```
(very minor, feel free to ignore; personally I prefer how you have it now but it's more consistent with the style of most of our headings)

Reply:
I referenced [The Data Catalog](https://kedro.readthedocs.io/en/0.18.2/data/data_catalog.html), but I see how it's not consistent. Maybe another opportunity for consistency across the board.